eng/common/pipelines/templates/steps/create-apireview.yml

```
@@ -2,28 +2,37 @@ parameters:
   ArtifactPath: $(Build.ArtifactStagingDirectory)
   Artifacts: []
   ConfigFileDir: $(Build.ArtifactStagingDirectory)/PackageInfo
+  MarkPackageAsShipped: $false
```

```suggestion
  MarkPackageAsShipped: false
```

Let's try not to mix the PS-isms in the Devops template.